test/libsolidity/util/SoltestTypes.h

```diff
@@ -286,12 +293,17 @@ struct FunctionCall
 		/// Marks a library deployment call.
 		Library,
 		/// Check that the storage of the current contract is empty or non-empty.
-		Storage
+		Storage,
+		/// Call to a builtin.
+		Builtin
 	};
 	Kind kind = Kind::Regular;
 	/// Marks this function call as "short-handed", meaning
 	/// no `->` declared.
 	bool omitsArrow = true;
 };
 
+using Builtin = std::function<std::optional<bytes>(FunctionCall const&)>;
+using Builtins = std::map<std::string, Builtin>;
```

Comment:
Please use an explicit type instead. The name sound like it would be a vector of Builtin.

Reply:
Ok, I removed `Builtins` now. True, it sounds more like it's a vector.